src/commons/sagas/LoginSaga.ts

```diff
@@ -9,12 +10,17 @@ import { showWarningMessage } from '../utils/notifications/NotificationsHelper';
 
 const LoginSaga = combineSagaHandlers({
   [SessionActions.login.type]: function* ({ payload: providerId }) {
-    const epUrl = computeEndpointUrl(providerId);
+    const isVscode = yield select(state => state.vscode.isVscode);
```

Comment:
```suggestion
    const isVscode = yield select(state => state.vscode?.isVscode);
```

All the tests failures would have been simply gone if I were to do this... but I'm not sure about this

Reply:
Why is the slice typed as optional?